Dockerfile

```diff
@@ -1,4 +1,88 @@
-FROM scratch AS production
+FROM busybox AS sct
```

Comment:
Let's call it something different, something that would describe what it is something it is doing different from other stages.